changelog.d/+releases-tests-fix.internal.md

```diff
@@ -0,0 +1 @@
+This is a fix for release tests
```

Comment:
This will land in the CHANGELOG.md file. This sentence doesn't fit as a changelog item.
But since there was no release since the first PR, this PR doesn't introduce a new change for the next version, and therefore doesn't necessitate a changelog entry.

So I'd recommend you delete the new changelog entry, and instead just make some small change to the changelog entry file you added in the first commit - just to pass the CI towncrier check.